### ISSUE TITLE
Email: Comments on WCAG 3.0 draft (4 - Mixing 3 sectors in one guideline)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#483]
[Comment from Email](https://lists.w3.org/Archives/Public/public-silver/2021Mar/0002.html): 
from [Word docx](https://lists.w3.org/Archives/Public/public-silver/2021Mar/att-0002/Review_Comments_for_WCAG_3.0_-_Vanderheiden.docx) 

3)  Mixing of three sectors in one guideline
There is a slight concern regarding mixing guidelines for three audiences in one document.  Will it confuse each audience with regard to which provisions apply to them.  But this is easily remedied by either organizing the document into sections – or tagging the provisions to indicate which apply to each group.  I think many will apply to all groups – and having the responsibilities of the different sectors all side by side is a feature of this document.   Provisions for that apply only to one audience, however, should be separate.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
